crs_motion_planning/src/motion_planning_server.cpp

```diff
@@ -220,6 +224,9 @@ class MotionPlanningServer : public rclcpp::Node
     trajopt_freespace_config.special_collision_cost.push_back({ "eoat_link", LOADED_PART_LINK_NAME, 0.03, 20.0 });
     trajopt_freespace_config.special_collision_cost.push_back({ "eoat_link", "robot_frame", 0.05, 15.0 });
     trajopt_freespace_config.special_collision_cost.push_back({ "shoulder_link", "robot_frame", 0.03, 15.0 });
+    trajopt_freespace_config.special_collision_cost.push_back({ "eoat_link", "table", 0.03, 20.0 });
```

Comment:
Same as above

Reply:
Is table specific to the swri setup?

Reply:
Table is swri specific, but won't negatively impact the system running in the other workcell. This will be accounted for in the config file I mentioned above.